nbs/18_callback.fp16.ipynb

```diff
@@ -770,8 +770,8 @@
     "class TestBeforeMixedPrecision(Callback):\n",
     "    order=-55\n",
     "    def before_fit(self): test_eq(first(self.model.parameters()).dtype, torch.float32)\n",
-    "    def before_batch(self): test_eq(self.x.dtype, torch.float32)\n",
-    "    def after_pred(self): test_eq(self.pred.dtype, torch.float16)\n",
+    "    def before_batch(self): test_eq(listify(self.x)[0].dtype, torch.float32)\n",
```

Comment:
Not sure we need to change the test do we?

Reply:
Yup, we do not need to change the tests.